cmd/config/validate_test.go

```diff
@@ -35,9 +35,9 @@ func validConfig() []byte {
 
 func invalidConfig() []byte {
 	config := v1beta1.DefaultClusterConfig()
-	config.Spec.Extensions.Storage.Type = "external_storage"
-	//  CreateDefaultStorageClass is not supported with external_storage
-	config.Spec.Extensions.Storage.CreateDefaultStorageClass = true
+	// NLLB cannot be used with external address
+	config.Spec.Network.NodeLocalLoadBalancing.Enabled = true
+	config.Spec.API.ExternalAddress = "k0s.example.com"
```

Comment:
hmm, is this really relevant for this PR? If removing openebs results this test being invalid/not-needed, maybe we should just remove the test?

Reply:
The test checks the config validation in general, it's not specific to OpenEBS, but used it as an example for an invalid config. So I just picked the first other invalid config combination that came to my mind.